spec/README.md

```diff
@@ -95,9 +99,11 @@ Updates to this specification will not reserve or assign meaning to
 any character "sigils" except for those in the `reserved` production.
 
 Updates to this specification
-will not remove any functions defined in the default registry nor
-will they remove any options or option values.
-Additional options or option values MAY be defined.
+will not remove any functions defined in the default registry.
+
+Updates to this specification will not remove any options or option values
+defined in the default registry.
+Additional options or additional option values for existing options MAY be defined.
```

Comment:
If we're this explicit about options & option values, we should note that new functions may be added.